src/transactions/feed/detailContent/DepositOrWithdrawContent.tsx

```diff
@@ -102,13 +110,16 @@ export function DepositOrWithdrawContent({ transaction }: DepositOrWithdrawConte
           feeType={FeeType.SecurityFee}
           transactionStatus={transaction.status}
         />
-        {transaction.swap && (
```

Comment:
Is this still needed for deposits or withdrawals without a swap?

Reply:
no, FeeItemRow will render null if no fee of type `AppFee` is found, which is only present for swaps